docs/getting-started/installation-and-upgrade/installation-references/tls-settings.md

```diff
@@ -23,3 +23,75 @@ The default TLS configuration only accepts TLS 1.2 and secure TLS cipher suites.
 |-----|-----|-----|-----|
 | `CATTLE_TLS_MIN_VERSION` | Minimum TLS version | `1.2` | `1.0`, `1.1`, `1.2`, `1.3` |
 | `CATTLE_TLS_CIPHERS` | Allowed TLS cipher suites | `TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`,<br/>`TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384`,<br/>`TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305`,<br/>`TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256`,<br/>`TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384`,<br/>`TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305` | See [Golang tls constants](https://golang.org/pkg/crypto/tls/#pkg-constants) |
+
+## Agent TLS Enforcement
+
+The `agent-tls-mode` setting controls how Rancher's agents (including cluster-agent, fleet-agent, and system-agent) evaluate Rancher's certificate.
+
+When set to `strict`, Rancher's agents will only trust certificates generated by the Certifcate Authority contained in the `cacerts` setting. 
+When set to `system-store`, Rancher's agents will trust any certificate generated by a public Certificate Authority contained in the OS' trust store (including Authorities such as Let's Encrypt). This can be a security risk, since any certificate generated by these authorities will be viewed as valid.
+
+While the `strict` option has a higher level of security, it requires Rancher to have access to the CA which generated the certificate visible to the agents. In the case of certain certificate configurations (notably, external certificates), this is not automatic, and extra configuration will be needed. See the [installation guide](../install-upgrade-on-a-kubernetes-cluster/install-upgrade-on-a-kubernetes-cluster.md#3-choose-your-ssl-configuration) for more information on which scenarios require extra configuration.
+
+This setting defaults to `strict` on new installs of Rancher (starting with v2.9.0), but is set as `system-store` for users installing/upgrading from a prior version.
+
+### Preparing for the Setting Change
+
+Each cluster contains a condition in the status field called `AgentTlsStrictCheck`. If this value is set to true, this indicates the agents for the cluster are ready to operate using the `strict` mode. You can manually inpsect each cluster to see if they are ready using the Rancher UI, or a Kubectl command like the following:
+
+```bash
+## the below command skips ouputs $CLUSTER_NAME,$STATUS for all non-local clusters
+kubectl get cluster.management.cattle.io -o jsonpath='{range .items[?(@.metadata.name!="local")]}{.metadata.name},{.status.conditions[?(@.type=="AgentTlsStrictCheck")].status}{"\n"}{end}'
+```
+
+### Changing the Setting
+
+You can change the setting using the Rancher UI or the `agentTLSMode` [helm chart option](./helm-chart-options.md).
+
+:::note
+
+If the value is set through the helm chart, it must be changed through the helm chart.
```

Comment:
```suggestion
If you specify the value using the Helm chart setting, you may modify it only with Helm.
```

Reply:
I left off "setting" from the first clause, since I think this is confusing when you bring the v3 setting into the mix.